pkg/capacity/capacity.go

```diff
@@ -107,6 +115,28 @@ var (
 		Factor:   1.1,
 		Steps:    10,
 	}
+
+	objectsGoalDesc = metrics.NewDesc(
+		"csistoragecapacities_desired_goal",
```

Comment:
nit: csistoragecapacities seems very long, do you think it makes sense to break them down a little bit like csi_storage_capacities_... totally up to you.

Reply:
I prefer `csistoragecapacities` for consistency with `kubectl get csistoragecapacities`.